build/openwisp_base/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.7-alpine AS BASE
+
+COPY ./openwisp_base/requirements.txt /opt/openwisp/requirements.txt
```

Comment:
don't you need to also install these dependencies? I would also move this command after the `RUN apk add --update --no-cache \ ...etc...` because the lines in `requirements.txt` may change more often than the system packages, so if we need to add one python package at least the system packages won't have to be rebuilt again.